core/forks.py

```diff
@@ -64,7 +67,10 @@ def auto_fork(section, input_category):
     port = cfg.get('port')
     username = cfg.get('username')
     password = cfg.get('password')
+    sso_username = cfg.get('sso_username')
```

Comment:
@echel0n 
Just checking if this still works for forks/sections where the `sso_username` and `sso_password` don't exist. i.e. do we need to have a fall-back to make this blank when the cfg.get returns an error?
I'm just reading and thinkign right now, will log in when I get home and do some tests...